lib/reporting/protocols_report.rb

```diff
@@ -318,14 +309,64 @@ def aal3_issuers_query
         | parse @message '"authn_context":[*]' as authn
         | filter
           name IN %{event}
-          AND (authn like /aal\\/3/ or acr like /aal\\/3/)
+          AND (authn like /ns\\/assurance\\/loa/ OR acr like /ns\\/assurance\\/loa/)
           AND properties.event_properties.success= 1
         | display issuer
         | sort issuer
         | dedup issuer
       QUERY
     end
 
+    def protocol_query
+      params = {
+        event: quote([SAML_AUTH_EVENT, OIDC_AUTH_EVENT]),
+      }
+
+      format(<<~QUERY, params)
+        fields
+          name AS protocol,
+          coalesce(properties.event_properties.service_provider, properties.event_properties.client_id) as issuer
+        | filter name IN %{event} AND properties.event_properties.success= 1
```

Comment:
1. I'd consider calling it "events" (plural) since it's a list
2. updating spacing around `success = 1`
```suggestion
        events: quote([SAML_AUTH_EVENT, OIDC_AUTH_EVENT]),
      }

      format(<<~QUERY, params)
        fields
          name AS protocol,
          coalesce(properties.event_properties.service_provider, properties.event_properties.client_id) as issuer
        | filter name IN %{events} AND properties.event_properties.success = 1
```

Reply:
sure, i'll fix as long as i'm in here https://github.com/18F/identity-idp/pull/11339/commits/e5d168559ef6cf72aa260b3046d1bbf82be121a4